src/sentry/static/sentry/app/actions/guideActions.jsx

```diff
@@ -0,0 +1,5 @@
+import Reflux from 'reflux';
+
+let GuideActions = Reflux.createActions(['fetchSuccess', 'guideClose', 'nextStep']);
```

Comment:
i know it's not that consistent, but I noticed we normally do verb before noun eg.` closeGuide`